spec/fixtures/CommonBuildSettings/configs/Objc_OSX_Framework/Objc_OSX_Framework_base.xcconfig

```diff
@@ -8,14 +9,16 @@ DYLIB_CURRENT_VERSION = 1
 DYLIB_INSTALL_NAME_BASE = @rpath
 ENABLE_STRICT_OBJC_MSGSEND = YES
 FRAMEWORK_VERSION = A
+GCC_NO_COMMON_BLOCKS = YES
 GCC_WARN_ABOUT_RETURN_TYPE = YES_ERROR
 GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE
 INFOPLIST_FILE = Objc_OSX_Framework/Info.plist
 INSTALL_PATH = $(LOCAL_LIBRARY_DIR)/Frameworks
 LD_RUNPATH_SEARCH_PATHS = $(inherited) @executable_path/../Frameworks @loader_path/Frameworks
-MACOSX_DEPLOYMENT_TARGET = 10.10
+MACOSX_DEPLOYMENT_TARGET = 10.11
+PRODUCT_BUNDLE_IDENTIFIER = me.segiddins.Objc-OSX-Framework
```

Comment:
For the sake of future diffs, it would be great to keep it consistent so the value doesn't change everytime someone else generates this file.